src/datagen/tinkers/java/novamachina/exnihilotinkers/tinkers/EXNTinkersToolDefinitions.java

```diff
@@ -46,7 +45,6 @@ protected void addToolDefinitions() {
                 .stat(ToolStats.ATTACK_SPEED, 1.2F)
                 .multiplier(ToolStats.DURABILITY, 2.2F)
                 .smallToolStartingSlots()
-                .action(ToolActions.PICKAXE_DIG)
```

Comment:
if i let this one in, cobblestone will be dropped instead of gravel when mining cobble. + gravel and sand won't dig faster so we don't need that either.